documentation/modules/operators/proc-configuring-multiple-cluster-operators.adoc

```diff
@@ -0,0 +1,122 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-using-multiple-cluster-operators.adoc
+
+[id='assembly-configuring-proxy-config-cluster-operator-{context}']
+= Configuring Cluster Operator replicas
+
+[role="_abstract"]
+To run additional Cluster Operator replicas in standby mode, you will need to increase the number of replicas and enable leader election.
+To configure leader election, use the leader election environment variables.
+
+To make the required changes, configure the following Cluster Operator installation files located in `install/cluster-operator/`: 
+
+* 060-Deployment-strimzi-cluster-operator.yaml
+* 022-ClusterRole-strimzi-cluster-operator-role.yaml
+* 022-RoleBinding-strimzi-cluster-operator.yaml
+
+Leader election has its own `ClusterRole` and `RoleBinding` RBAC resources that target the namespace where the Cluster Operator is running, rather than the namespace it is watching. 
+
+The default deployment configuration creates a `Lease` resource called `strimzi-cluster-operator` in the same namespace as the Cluster Operator.
+The Cluster Operator uses leases to manage leader election. 
+The RBAC resources provide the permissions to use the `Lease` resource.
+If you use a different `Lease` name or namespace, update the `ClusterRole` and `RoleBinding` files accordingly.
+
+.Prerequisites
+
+* This procedure requires use of a Kubernetes user account which is able to create `CustomResourceDefinition`, `ClusterRole`, and `ClusterRoleBinding` resources.
+Use of Role Base Access Control (RBAC) in the Kubernetes cluster usually means that permission to create, edit, and delete these resources is limited to Kubernetes cluster administrators, such as `system:admin`.
```

Comment:
I would probably rephrase it to _you need account with privileged to manage these resources. I would leave the cluster or system admin our of it.

Reply:
Changed to:

> You need an account with permission to create and manage `CustomResourceDefinition` and RBAC (`ClusterRole`, and `RoleBinding`) resources.

Reply:
@scholzj -- We use the same prereq in the Deploying guide for deploying the Cluster Operator. Should we reduce down there too? I don't think we lose anything